trinity/main.py

```diff
@@ -297,7 +309,6 @@ def kill_trinity_gracefully(logger: logging.Logger,
     time.sleep(0.2)
     kill_process_gracefully(networking_process, logger)
     logger.info('Networking process (pid=%d) terminated', networking_process.pid)
-    sys.exit()
```

Comment:
Caution! That one was introduced to support programatically calling `killl_trinity_gracefully` (as opposed to hitting it through `CTRL+C`).

https://github.com/ethereum/py-evm/blob/ca054545c4afea5b0961c849486b189daa750fff/trinity/main.py#L245-L254

If you remove this, you'll notice that `trinity --tx-pool --light` will leave you with a hanging process.